### PR TITLE
change base image to eclipse-temurin:11-jdk

### PR DESCRIPTION
**Pull request by yaskoo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-    cd ${GRAPHDB_PARENT_DIR} && \
-    unzip /tmp/graphdb-${version}.zip && \
-    rm /tmp/graphdb-${version}.zip && \
-    mv graphdb-${version} dist && \
-    mkdir -p ${GRAPHDB_HOME}
+RUN <<EOF
+	apt-get update
+	apt-get install -y net-tools less unzip
+	curl -fsSL "${download_url}" > graphdb-${version}.zip
+	bash -c 'md5sum -c - <<<"$(curl -fsSL ${download_url_checksum})  graphdb-${version}.zip"'
```
Comment (mihailradkov) [inline Dockerfile:17]: Is `bash -c` needed ?

File: Makefile [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+endif
+ifdef ARCH
+TAG := ${TAG}-${ARCH}
+endif
 
-build-image-arm64:
-	docker build --no-cache --pull --build-arg version=${VERSION} -t ontotext/graphdb:${VERSION}-arm64 arm64-build-on-aws
+temurin:
+	curl ${DOCKERFILE_BASE} > Dockerfile.base
+	sed -i '/^COPY/d;/^ENTRYPOINT/d' Dockerfile.base
```
Comment (mihailradkov) [inline Makefile:16]: Maybe document what is going on here?

Comment (yaskoo) [reply Makefile:17]: done

File: Dockerfile (hunk 1 above)
Comment (yaskoo) [reply Dockerfile:17]: tl; dr; yes

Default shell is `sh` which doesn't support the herestring (`<<<`) operator. We could use other approaches like changing the shell, pipes, writing to a file, but that will complicate things further.

**Review by mihailradkov — APPROVED** [review]

**Pull request closed without merging**